components/usage/pkg/db/varchar_test.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package db_test
```

Comment:
These tests were moved to a seperate file because they required using the `dbtest.ConnectForTest` but were previously in the `db` package. This would create a circular import. Nothing changed about the test.